extensions.json

```
@@ -1184,21 +1182,19 @@
     {
       "id": "ms-pyright.vscode-pyright",
       "repository": "https://github.com/Microsoft/pyright",
-      "version": "1.1.159",
-      "checkout": "1.1.159",
+      "version": "1.1.163",
+      "checkout": "1.1.163",
       "location": "packages/vscode-pyright",
       "prepublish": "cd packages/vscode-pyright && npm run prepackage"
     },
     {
       "id": "ms-python.python",
-      "download": "https://github.com/microsoft/vscode-python/releases/download/2021.7.1060902895/ms-python-release.vsix",
-      "version": "2021.7.1060902895"
+      "download": "https://github.com/microsoft/vscode-python/releases/download/2021.8.1147840270/ms-python-release.vsix",
+      "version": "2021.8.1147840270"
     },
     {
       "id": "ms-toolsai.jupyter",
       "repository": "https://github.com/microsoft/vscode-jupyter",
```

Let's keep the latest version pinned:

```suggestion
      "repository": "https://github.com/microsoft/vscode-jupyter",
      "version": "2021.8.12",
      "checkout": "2021.8.12",
```